custom/icds/tests/test_get_icds_sms_usage.py

```diff
@@ -0,0 +1,210 @@
+            make_loc('state2', type='state', domain=cls.domain),
+        )
+        cls.districts = (
+            make_loc('district', type='district', domain=cls.domain, parent=cls.states[0]),
+            make_loc('district2', type='district', domain=cls.domain, parent=cls.states[1]),
+        )
+        cls.blocks = (
+            make_loc('block', type='block', domain=cls.domain, parent=cls.districts[0]),
+            make_loc('block2', type='block', domain=cls.domain, parent=cls.districts[1]),
+        )
+        cls.supervisors = (
+            make_loc('supervisor', type='supervisor', domain=cls.domain, parent=cls.blocks[0]),
+            make_loc('supervisor2', type='supervisor', domain=cls.domain, parent=cls.blocks[1]),
+        )
+        cls.awcs = (
+            make_loc('awc', type='awc', domain=cls.domain, parent=cls.supervisors[0]),
+            make_loc('awc2', type='awc', domain=cls.domain, parent=cls.supervisors[1]),
+        )
+        cls.users = (
+            make_user('user', cls.awcs[0]),
+            make_user('user2', cls.awcs[1]),
+        )
+
+        cls.sms_list.append(SMS.objects.create(
+            domain=cls.domain,
+            date=date(2017, 4, 10),
+            backend_api=AirtelTCLBackend.get_api_id(),
+            direction='O',
+            processed=True,
+            couch_recipient=cls.users[0]._id,
+            custom_metadata={'icds_indicator': 'xxx'}
+        ))
+        cls.sms_list.append(SMS.objects.create(
+            domain=cls.domain,
+            date=date(2017, 4, 10),
+            backend_api=AirtelTCLBackend.get_api_id(),
+            direction='O',
+            processed=True,
+            couch_recipient=cls.users[0]._id,
+            custom_metadata={'icds_indicator': 'xxx'}
+        ))
+        cls.sms_list.append(SMS.objects.create(
+            domain=cls.domain,
+            date=date(2017, 4, 10),
+            backend_api=AirtelTCLBackend.get_api_id(),
+            direction='O',
+            processed=True,
+            couch_recipient=cls.users[1]._id,
+            custom_metadata={'icds_indicator': 'xxx'}
+        ))
+        cls.sms_list.append(SMS.objects.create(
+            domain=cls.domain,
+            date=date(2017, 4, 10),
+            backend_api=AirtelTCLBackend.get_api_id(),
+            direction='O',
+            processed=True,
+            couch_recipient=cls.users[1]._id,
+            custom_metadata={'icds_indicator': 'aaa'}
+        ))
+        date_start = '2017-04-01'
+        date_end = '2017-05-01'
+        call_command("get_icds_sms_usage", cls.domain, date_start, date_end)
+        cls.workbook_name = 'icds-sms-usage--{0}--{1}.xlsx'.format(date_start, date_end)
+        cls.workbook = openpyxl.load_workbook(cls.workbook_name)
+
+    @classmethod
+    def tearDownClass(cls):
+        super(GetICDSSmsUsageTest, cls).tearDownClass()
+        cls.workbook.close()
+        os.remove(cls.workbook_name)
```

Comment:
Should delete the locations, users, and `SMS` objects that were created in setup.